recipe/meta.yaml

```diff
@@ -3,14 +3,14 @@
 
 # https://github.com/apache/arrow/blob/master/cpp/src/parquet/.parquetcppversion
 {% set parquet_version = "1.5.1" %}
-{% set arrow_version = "0.11.0" %}
+{% set arrow_version = "0.11.1" %}
```

Comment:
I think this should not be pinned. Can you change this to a `>=` condition? Ideally we will not be altering this package any more

Reply:
Indeed! Done.